### PR TITLE
Show image in full screen

### PR DESCRIPTION
**Pull request by MSamiTariq**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [ ] I have carefully reviewed my code and believe it is final from my side.
- [ ] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [ ] I have tested the new feature or updates in both light and dark themes.
- [ ] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [ ] I have tested my changes on mobile screen sizes.
- [ ] I have tested my changes on tablet screen sizes.
- [ ] I have tested my changes on laptop screen sizes.

### Functional Testing
- [ ] I have thoroughly tested the functionality of my changes.
- [ ] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [ ] Explain the task/bug you are solving
- [ ] Show the changes in the UI
- [ ] Show the changes in the codebase
- [ ] Show the changes you tested to make sure it works
- [ ] Show the changes in various screen sizes
- [ ] Show the changes in both light and dark themes

### CODE REVIEW

File: shared/web-core/src/components/core/modals/FullScreenModal.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const StyledModalWrapper = styled.div`
 export default function FullScreenModal({ open, title, children, onClose, showCloseButton = true, showTitleBg = true }: SingleSectionModalProps) {
   return (
     <Transition.Root show={open} as={Fragment}>
-      <Dialog as="div" className="relative z-10" onClose={onClose}>
+      <Dialog as="div" className="relative z-30" onClose={onClose}>
```
Comment (DawoodMehmood) [inline shared/web-core/src/components/core/modals/FullScreenModal.tsx:23]: what was the problem here?

Comment (MSamiTariq) [reply shared/web-core/src/components/core/modals/FullScreenModal.tsx:23]: We have the z-index as z-20 on the FullScreenImage:
`<div className="absolute left-20 right-20 bottom-20 top-20 z-20">`

So the full screen modal which is used to show the enlarged image was getting placed under this original image. Just to bring it above, I've increased the z-index
